### DISCUSSION TITLE
How to automatically download the pre-release version

### DISCUSSION THREAD

**Discussion by vipwuping** [AdguardTeam/AdGuardHome#3402] [Q&A]
https://github.com/AdguardTeam/AdGuardHome/releases/download/${latest_ver}/AdGuardHome_linux_${Arch}.tar.gz
I now use this method to download the latest version
How to automatically download the pre-release version

**Comment by ameshkov** [comment]
You can use the URLs listed here:
https://github.com/AdguardTeam/AdGuardHome#test-unstable-versions
